docs/management_api.yml

```diff
@@ -70,6 +70,19 @@ paths:
             - inprogress
             - finished
             - pending
+        - name: type
+          in: query
+          description: |
+              Deployment type filter.
+              It is possible for the deployment to not have the type field.
+              In this case, the deployments service will treat it
+              as software deployment and return it with other
+              software deployments.
```

Comment:
This is an internal implementation detail, I don't think we need to expose it here, do we?
In particular, we have to be sure that if you do not provide a type value, you are getting all deployments. If you set "software", you only retrieve software deployments, including also the ones which don't have the value set in the database.

Reply:
I agree that this is an implementation detail, but on the other hand, we will return to the users deployments with type=software, deployments with type=configuration, and deployments without type filed; I thought it may be confusing and it would be nice to document it somewhere; 
I don't have a strong opinion here so I can remove this comment if you want

Reply:
done

Reply:
My idea was that we never return objects with the empty value: if we don't have a value in the db, we set "software" when we marshall to JSON. From the user's point of view, it's always either software or configuration, he will never see empty.